src/web/lib/getCommentPage.ts

```diff
@@ -0,0 +1,38 @@
+import { joinUrl } from '@root/src/web/lib/joinUrl';
+
+// An example /context response
+// {
+//     status: 'ok',
+//     commentId: 3519111,
+//     commentAncestorId: 3519111,
+//     discussionKey: '/p/27y27',
+//     discussionWebUrl:
+//         'https://www.theguardian.com/commentisfree/cifamerica/2009/may/14/washington-post-torture-libel',
+//     discussionApiUrl:
+//         'https://discussion.guardianapis.com/discussion-api/discussion//p/27y27?orderBy=oldest&pageSize=20&page=1',
+//     orderBy: 'oldest',
+//     pageSize: 20,
+//     page: 1,
+// };
+
+export const getCommentPage = async (
```

Comment:
Why is this not in the API file?

Reply:
There is no api file 🤷‍♂ 

The file with the name `api` is probably missnamed and should be called `useApi`